### DISCUSSION TITLE
Drag and drop

### DISCUSSION THREAD

**Discussion by maniactwenty** [Tyrrrz/YoutubeDownloader#379] [Q&A]
Hello,
Foremost, thanks to Tyrrz and all other contributors for bringing this tool as most productive as it must be.
My question is, is there any chance to add a Drag and drop option to the tool, so it can be even more productive?
Thanks in advance.

**Comment by Tyrrrz** [comment]
Hi. What are you trying to drag and drop?

**Reply by maniactwenty** [reply]
I tried drag the downloaded videos into Premiere or AE directly without going to the file folder, but it didn't work.

**Reply by Tyrrrz** [reply]
Ah I see what you mean

**Reply by maniactwenty** [reply]
Appreciate it, and I hope you make it as a feature the next updates. Thank you.
